test/index.spec.js

```diff
@@ -91,7 +91,7 @@ describe('FsDatastore', () => {
   })
 
   it('query', (done) => {
-    const fs = new FsStore(path.join(__dirname, 'test-repo/blocks'))
+    const fs = new FsStore(path.join(__dirname, 'test-repo', 'blocks'))
```

Comment:
nice, but not needed, `path.join` actually takes care of the normalisation already. 

```js
path.join('a/b', 'c') === path.join('a', 'b', 'c')
```

Reply:
Yes you are right.  I was just being paranoid about slashes when trying to get it working on windows.

Reply:
Leaving out the platform-specific dividers makes it easier for everyone on platforms that don't use forward-slashes though.